README.md

```diff
@@ -1,38 +1,132 @@
-
-
-```yaml
-name: Lint CloudFormation Templates
-
-on: [push]
-
-jobs:
-  cloudformation-linter:
-
-    runs-on: ubuntu-latest
-
-    steps:
-    - name: Checkout
-      uses: actions/checkout@v2
-
-    - name: cfn-lint
-      uses: docker://ghcr.io/scottbrenner/cfn-lint-action:master
-      with:
-        args: "**/*.yaml"
-```
-
-See [Basic Usage](https://github.com/aws-cloudformation/cfn-python-lint#basic-usage) for full usage details.
-
-## License
-
-The Dockerfile and associated scripts and documentation in this project are released under the [MIT License](LICENSE).
-
-Container images built with this project include third party materials. See [THIRD_PARTY_NOTICE.md](THIRD_PARTY_NOTICE.md) for details.
+# GitHub Action for CloudFormation Linter
+
+This Action for [CloudFormation Linter](https://github.com/aws-cloudformation/cfn-python-lint/) enables arbitrary actions for interacting with CloudFormation Linter to validate CloudFormation yaml/json templates against the CloudFormation spec and additional checks. Includes checking valid values for resource properties and best practices.
+
+The Action's primary purpose is to set the [CloudFormation Linter](https://github.com/aws-cloudformation/cfn-python-lint/) to the runner's $PATH so you can access the linter throughout the workflow, and use in a way that suits your application. There is a way to run a [CloudFormation Linter](https://github.com/aws-cloudformation/cfn-python-lint/) command in this workflow though if you like.
+
+## Usage
+
+There are multiple ways to consume this GitHub Action.
+
+The recommended approach would be to to use this action to add the [CloudFormation Linter](https://github.com/aws-cloudformation/cfn-python-lint/) to the runners $PATH and run commands throughout the rest of the workflow. An example of that can be found below:
+
+```yaml
+name: Lint CloudFormation Templates
+
+on: [push]
+
+jobs:
+  cloudformation-linter:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Cloud Formation Linter with Latest Version
+        uses: scottbrenner/cfn-lint-action@v2
+
+      - name: Print the Cloud Formation Linter Version & run Linter.
+        run: |
+          cfn-lint --version
+          cfn-lint -t ./template.yml
+```
+
+Within the `run |` section, you specify the required Cloud Formation Linter commands.
+
+However, if you would rather run the Cloud CloudFormation Linter commands within this action. That is also possible. An example of that can be found below:
+
+```yaml
+name: Lint CloudFormation Templates
+
+on: [push]
+
+jobs:
+  cloudformation-linter:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Testing with CFN Lint Command
+        uses: ./
```

Comment:
```suggestion
        uses: scottbrenner/cfn-lint-action@v2.0.0
```
This probably needs to be the Action, right? Don't think `./` will work outside of this repository.

Reply:
BOOM 👍 correct! See my comment above about the v2 thing @ScottBrenner but I agree this should be: `scottbrenner/cfn-lint-action@*` yes! Thanks for the good find 👍